README.md

```diff
@@ -163,60 +163,68 @@ This method generates the following:
+Qemistree will use `ms2_smiles` to make chemical taxonomy assignments, when MS2 matches are available for a feature. Otherwise, `csi_smiles` will be used. The column `structure_source` in `classified-merged-feature-data.qza` records whether taxonomic assignment was done using CSI:FingerID predictions or MS/MS library matches.
+Lastly, Qemistree includes some utility functions that are useful to visualize and explore the molecular hierarchy generated above.
 
 1. Prune molecular hierarchy to keep only the molecules with annotations.
 
 ```bash
 qiime qemistree prune-hierarchy \
   --i-feature-data classified-merged-feature-data.qza \
-  --p-column smiles \
+  --p-column class \
   --i-tree merged-qemistree.qza \
-  --o-pruned-tree merged-qemistree-smiles.qza
+  --o-pruned-tree merged-qemistree-class.qza
 ```
 
-Users can choose any of the data columns (`--p-column`) that are in the `classified-merged-feature-data.qza` file for pruning. For e.g. '#FeatureID','kingdom', 'superclass', 'class', 'subclass', 'direct_parent', and 'smiles'. All features with no data in this column will be removed from the phylogeny. **Note:** pruning by '#FeatureID' will not remove any of the features as they all should have this form of annotation. The use of this columns becomes useful for representing unclassified features on the tree.
+Users can choose any of the data columns (`--p-column`) that are in the `classified-merged-feature-data.qza` file to prune the hierarchy. For e.g. '#featureID','kingdom', 'superclass', 'class', 'subclass', 'direct_parent', and 'smiles'. All features with no data in this column will be removed from the phylogeny.
 
-2. Generate an annotated qemistree tree in [iTOL](https://itol.embl.de/).
+2.1 Generate an annotated qemistree tree in [iTOL](https://itol.embl.de/).
 
-If the user has groups and/or conditions by which they want to visually compare the features, a grouped table file can be inputted to the iTOL tree that contains all the summarize information for each feature stratified by the group/condition. This will generate normalized barcharts at the tips of the tree specifying the relative abundance of the feature in each particular group/condition the feature is found in. 
+```bash
+qiime qemistree plot \
+  --i-tree merged-qemistree-class.qza \
+  --i-feature-metadata classified-merged-feature-data.qza \
+  --p-category class \
+  --o-visualization /path-to-qemistree-plot.qzv/
+```
+This QZV can be visualized in [iTOL](https://itol.embl.de/) using [Qiime2 Viewer](https://view.qiime2.org); iTOL interface can be used to interact and make visual modifications.
+
+2.2 Add sample metadata to tree tips
+
+If the user has sample metadata columns by they want to compare samples, Qemistree enables them to visualize feature abundance barcharts at the tips of the tree (abundance or relative abundance) of the feature stratified by the sample metadata column of interest. This can be done as follows:

```

Comment:
```suggestion
If the user has sample metadata columns to compare groups of samples, Qemistree enables them to visualize feature abundance barcharts at the tips of the tree (abundance or relative abundance) of the feature stratified by the sample metadata column of interest. This can be done as follows:

```